.eslintrc.js

```diff
@@ -43,7 +43,21 @@ module.exports = {
         'no-console': ['warn', { allow: ['warn', 'error'] }],
         semi: ['error', 'never'],
         'import/order': 'off',
-        'import/extensions': 'off',
+        'import/extensions': ['error', 'ignorePackages'],
```

Comment:
Do you know if this is autofixeable?

Reply:
I don't think it is. The only way to get something like that is to convert all of the packages to use moduleResolution: NodeNext, and then TypeScript will suggest imports with .js endings from the start.

I can try that, if you all are open to it. It's a bigger change / commitment.

Reply:
I briefly tried locally, and it creates a bunch of TS type resolution issues that need to be resolved. The project builds, but there are lots of type issues that appear in the editor.

Reply:
If you attempt this after merging this, see ProseMirror/prosemirror#1305, where we discussed issues here in depth. In order to build with NodeNext, here's what I did to start:

1. Update main tsconfig.json moduleResolution to "NodeNext"
2. Override moduleResolution in tests/tsconfig.json to be "node", so cypress tests still build/run
3. Update all prosemirror packages to get the compatible dependencies

Reply:
@bdbch is auto-fixability a blocker?

Reply:
Not really. I just need another review from @svenadlung since we share PR responsibilities.

Reply:
Heads up that there is now the `typescript.preferences.importModuleSpecifierEnding` for VSCode, which will make .js endings the default when auto-importing!